CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Changelog
 
+## 1.0.0-BETA20
+* Add cursor optional functions: `getStringOptional`, `getLongOptional`, `getDoubleOptional`, `getBooleanOptional` and `getBytesOptional` when using the column name which allow for optional return types
+* Throw errors for invalid column on all cursor functions
+* `getString`, `getLong`, `getBytes`, `getDouble` and `getBoolean` used with the column name will now throw an error for non-null values and expect a non optional return type
```

Comment:
Is this a breaking change from the previous `BETA19` release?

Reply:
Yes. We haven't officially released BETA19 so I figured it did not require a mention.